src/retention/policy/fifo_10_up_to_7_months.py

```diff
@@ -4,17 +4,17 @@
     GracePeriodAfterDeletionFilter
 from src.retention.policy.filter.most_recent_daily_backup_filter import \
     MostRecentDailyBackupFilter
-from src.retention.policy.filter.ten_backup_versions_filter import \
-    TenBackupVersionsFilter
-from src.retention.policy.filter.younger_than_7_months_filter import \
-    YoungerThan7MonthsFilter
+from src.retention.policy.filter.ten_young_backup_versions_filter import \
+    TenYoungBackupVersionsFilter
+from src.retention.policy.filter.only_one_version_above_7_months_filter import \
+    OnlyOneVersionAbove7MonthsFilter
 
 
 class Fifo10UpTo7Months(object):
```

Comment:
Please rename as retention strategy is now more customised

Reply:
renamed to RetentionPolicy. done